.ci/magician/cmd/generate_comment.go

```diff
@@ -295,35 +305,6 @@ func execGenerateComment(prNumber int, ghTokenMagicModules, buildId, buildStep,
 		errors["Other"] = append(errors["Other"], "Failed to update breaking-change status check with state: "+breakingState)
 	}
 
-	// Run missing test detector (currently only for beta)
-	missingTestsPath := mmLocalPath
-	for _, repo := range []source.Repo{tpgbRepo} {
-		if !repo.Cloned {
-			fmt.Println("Skipping missing tests; repo failed to clone: ", repo.Name)
-			continue
-		}
-		missingTests, err := detectMissingTests(missingTestsPath, repo.Path, oldBranch, rnr)
-		if err != nil {
-			fmt.Println("Error running missing test detector: ", err)
-			errors[repo.Title] = append(errors[repo.Title], "The missing test detector failed to run.")
-		}
-		data.MissingTests = missingTests
-	}
-
-	// Run unit tests for missing test detector
-	if err = runMissingTestUnitTests(
```

Comment:
It looks like this PR will make us rely on the diff processor unit tests, which run using the main branch of TPG (rather than the current branch of TPGB.) However, we're still running the actual detector against the current branch.

We should probably run diff processor unit tests on TPGB as well. Do we also need to run missing test detector tests here still? That is, are there any issues that `TestReadAllTests` would catch that would _only_ be caught if it were run against the auto-pr branch?